pkg/config/identifier.go

```diff
@@ -316,6 +316,7 @@ func IsArchive(id string) Archive {
 // Clear clears loc and mimeinfo details to avoid pollution when creating multiple identifiers in same session
 func Clear() func() private {
 	return func() private {
+		identifier.noContainer = false
```

Comment:
Apparently we need a change like this, but we're not then resetting reports to `pronom` so I am not sure which way is correct of if there is still some pollution somewhere?

Reply:
resetting reports to `pronom` probably a good idea too. In principle I'd say it's good to add anything to Clear where you are setting config options that change defaults.